containers/agent/checksum_discovery.go

```diff
@@ -128,7 +128,7 @@ func runChecksumStartCmd(_ *cobra.Command, _ []string) {
 	}
 
 	if checksumFileName == "" {
-		panic(fmt.Errorf("could not find any OSImage checksum file in %s", checksumStartOpts.osImageDir))
+		panic(fmt.Errorf("%w in %s", ErrOSImageNotFound, checksumStartOpts.osImageDir))
```

Comment:
Hmm.. Does `err113` linter crib only for this one? We use it everywhere[1]. Realistically neither we compare the error anywhere nor there is memory related optimization, as it's used only once.

[1] https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fopenstack-baremetal-operator%20fmt.Errorf&type=code